tags/pickMonster.tag

```diff
@@ -0,0 +1,92 @@
+<pickmonster>
+  <!-- the image select check is based of Birjitsinh-->
+  <!-- code found in https://bootsnipp.com/snippets/Zl6ql -->
+
+  <!-- HTML -->
+  <!-- Based of Birjitsinh code -->
+  <div class="container">
+    <h1 class="display-3 text-center">What monsters might help you on your journey?</h1>
+    <div class="row">
+      <form method="get">
+        <div class="form-group">
+          <div class="col-md-4" each={ monsterItem, i in myMonsters}>
+            <label class="btn btn-info">
+              <img src={ monsterItem.img } alt={ monsterItem.name } class="img-thumbnail img-check { check: monsterItem.pick }" onclick={ parent.toggle }>
+              <input type="checkbox" name={ monsterItem.name } id={ monsterItem.id } class="hidden">
+            </label>
+          </div>
+        </div>
+      </div>
+      <button type="button" class="btn btn-success" onclick={ selectMonster }>Next</button> <!-- need to add link to next page -->
+    </form>
+  </div>
+</div>
+
+<script>
+  // JAVASCRIPT
+  var that = this;
+  let database = firebase.firestore()
+  var monsterRef = database.collection('Monsters');
+  this.myMonsters = [];
+  let refPredictedMonsters;
+
+  //read monster assets from database
+  monsterRef.onSnapshot(function (snapshot) {
+    var monsters = [];
+    snapshot.forEach(function (doc) {
+      this.monster = true;
+      monsters.push(doc.data());
+    })
+    that.myMonsters = monsters;
+    that.update();
+  });
+
+  // receives projectId
+  observer.on('project:created', (curProject) => {
+    this.projectId = curProject;
```

Comment:
OH I didn't understand how observers worked until now! this is great :)

Reply:
Yay! <3